examples/angular/basic-external-state/src/app/app.html

```
@@ -0,0 +1,106 @@
+        </tr>
+      }
+    </tbody>
+  </table>
+  <div class="spacer-sm"></div>
+  <div class="controls">
+    <button
+      class="demo-button demo-button-sm"
+      (click)="table.setPageIndex(0)"
+      [disabled]="!table.getCanPreviousPage()"
+    >
+      <<
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)